_components/banner/guidance/implementation.md

```diff
@@ -1,3 +1,7 @@
 - Some .mil websites do not belong to an official U.S. Department of Defense organization. These sites should adapt the **Official websites use .mil section** to use more technically accurate language: “A **.mil** website operates under the approval of the U.S. Department of Defense.”
 - The banner should directly follow the `skipnav` component.
 - Set the banner background color with `$theme-banner-background-color`. Banner text will update automatically.
+
+{:.site-note}
+**Note:** We recommend using `uswds-init` when using `banner` to assist in preventing flashes of unstyled content as well as cumulative layout shift (CLS). This small JavaScript file (less than 1 KB minified) helps the browser know if the USWDS JavaScript library is loading properly.
```

Comment:
If CLS gets an abbreviation it makes sense to add it to FOUC.

Reply:
Resolved in [1a72480](https://github.com/uswds/uswds-site/pull/1888/commits/1a724800888d5d1084544376675c923e5ab442d1)